index.html

```diff
@@ -1,164 +1,197 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="popup.css" />
 
-  <section id="detail1">
-    <button type="button" id="modal-close">X</button>
-    <div id="detail"></div>
-   
+    <title>John Mutisya's Portofolio</title>
+  </head>
+  <body>
+    <section id="detail1">
+      <button type="button" id="modal-close">X</button>
+      <div id="detail"></div>
     </section>
-  <div class="blurBg">
+    <div class="blurBg">
+      <section class="main-wrapper">
+        <header>
+          <nav class="navbar">
+            <a href="#" class="nav-left">John Mutisya</a>
+            <ul class="menu-list">
+              <li class="menu-list-items">
+                <a class="menu-list-link" href="#Portofolio">Portofolio</a
+                ><span class="mobile-span"> > </span>
+              </li>
+              <li class="menu-list-items">
+                <a class="menu-list-link" href="#About">About</a>
+                <span class="mobile-span"> > </span>
+              </li>
+              <li class="menu-list-items">
+                <a class="menu-list-link" href="#Contact">Contact</a>
+                <span class="mobile-span"> ></span>
+              </li>
+              <li class="menu-list-items">
+                <img
+                  class="menu-list-link mail-icon"
+                  src="./images/mail-icon-removebg-preview.png"
+                  alt=""
+                />
+              </li>
+            </ul>
 
-<section class="main-wrapper">
-  <header>
-    <nav class="navbar">
-      <a href="#" class="nav-left">John Mutisya</a>
-      <ul class="menu-list">
-        <li class="menu-list-items"><a class="menu-list-link" href="#Portofolio">Portofolio</a><span class="mobile-span"> > </span></li>
-        <li class="menu-list-items"><a class="menu-list-link" href="#About">About</a> <span class="mobile-span"> > </span> </li>
-        <li class="menu-list-items"><a class="menu-list-link" href="#Contact">Contact</a> <span class="mobile-span"> ></span> </li>
-        <li class="menu-list-items"><img class="menu-list-link mail-icon" src="./images/mail-icon-removebg-preview.png" alt=""></li>
-      </ul>
-      
-      <div class="hamberg">
-        <!-- <img src="images/menu_icon.svg" alt="menu icon image"> -->
-        <span class="bar"></span>
-        <span class="bar"></span>
-        <span class="bar"></span>
-      </div>
-   
-    </nav>
-  </header> 
+            <div class="hamberg">
+              <!-- <img src="images/menu_icon.svg" alt="menu icon image"> -->
```

Comment:
- [ ] To make clean code kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=Follow%20our%20list%20of%20best%20practices%20for%20HTML%20%26%20CSS.)